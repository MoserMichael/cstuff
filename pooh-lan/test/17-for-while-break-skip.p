# for loop ; test keywords: break - finishing the iteration and skip - continue to next iteration. 

println( ~msg 'while loop with skip and break' )

i = 1
while i < 11
  if i == 10 
    break
  end    
  if i % 2 == 0
    i = i + 1
    skip
  end
  println( ~msg i )
  i = i + 1
end  

println( ~msg 'for loop with skip and break' )

for i range( ~from 1 ~to 11 )
  if i == 10 
    break
  end    

  if i % 2 == 0
    skip
  end
  println( ~msg i )
end

println( ~msg '*** eof ***' )
