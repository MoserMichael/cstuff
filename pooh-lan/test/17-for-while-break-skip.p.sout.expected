while loop with skip and break
1
3
5
7
9
for loop with skip and break
1
3
5
7
9
*** eof ***
